### DISCUSSION TITLE
time series

### DISCUSSION THREAD

**Discussion by darbary** [biolab/orange3#5185] [Q&A]
In time series, i down loaded in  Yahoo Finance, ^DJI, connected it to VAR Model & then to Model Evaluation, but i got a blank window instead of RMSE, etc. . What did i do wrong?

**Comment by ajdapretnar** [comment]
A screenshot of the workflow would help.

**Comment by darbary** [comment]
![image](https://user-images.githubusercontent.com/67218187/104601758-9e912300-56a0-11eb-8ca5-cef2534fca05.png)

**Reply by darbary** [reply]
thanks

**Comment by ajdapretnar** [comment]
As with any model evaluation, scoring requires predictions and the data. Same goes for time series. So the right workflow would look like:
<img width="383" alt="Screen Shot 2021-01-14 at 15 15 24" src="https://user-images.githubusercontent.com/12524972/104602244-7c38de80-567b-11eb-946d-5e4ad63d5773.png">

This example is missing in the documentation and should be added.

**Comment by darbary** [comment]
![image](https://user-images.githubusercontent.com/67218187/104602778-c339ca80-56a1-11eb-9c31-e8a47e3969dc.png)

**Comment by darbary** [comment]
tried it but still missing, thanks for your super fast responce

**Comment by ajdapretnar** [comment]
Did you press Apply?

**Reply by darbary** [reply]
thanks a million, got it now
